### ISSUE TITLE
3D bubble charts not working in Firefox

### ISSUE THREAD

**Issue by alubbock** [plotly/plotly.js#2597]
Bubbles are not appearing on the 3D bubble charts in Firefox 59.0.2 on macOS 10.13.4 (see screenshot below, taken from the first chart on https://plot.ly/python/3d-bubble-charts/), however the mouseover labels still work. The charts appear to work fine on Chrome and Safari.

<img width="739" alt="Bubble chart without bubbles" src="https://user-images.githubusercontent.com/2590257/39482577-375716e4-4d35-11e8-975c-6345d802c473.png">

**Comment by etpinard** [comment]
Strange. This is working fine on Firefox 59.0.2 on my Ubuntu 16.04 machine.

@alubbock a couple questions:

- do you any console errors?
- can you provide some info about your computer's hardware?

Thanks for the report!

**Comment by alubbock** [comment]
@etpinard I get the following console messages, but nothing that looks like an error (copy/pasted with line numbers included):

```
new-to-plotly
3d-bubble-charts:24556:9
simple-bubble-chart
3d-bubble-charts:24556:9
bubble-chart-sized-by-a-variable
3d-bubble-charts:24556:9
edit-the-colorbar
3d-bubble-charts:24556:9
reference
3d-bubble-charts:24556:9
WARNING: cdn.mathjax.org has been retired. Check https://www.mathjax.org/cdn-shutting-down/ for migration tips.
MathJax.js:32:5
```

I'm using a 2014 Macbook Pro with Intel Iris Pro + NVIDIA GT 750M graphics.

**Comment by alexcjohnson** [comment]
On my mac the bubbles show up but the tick labels are broken - also only in FF
![screen shot 2018-05-01 at 6 41 38 pm](https://user-images.githubusercontent.com/2678795/39497034-53e2a0b4-4d6f-11e8-9242-5eedd8e68c7d.png)
Firefox 59.0.2
2015 MacBook Pro with Intel Iris 6100 1536MB

**Comment by etpinard** [comment]
Both bubbles and tick labels appear to render ok (at least on browserstack):

![image](https://user-images.githubusercontent.com/6675409/54853562-9cd5be80-4cc6-11e9-8087-7cde3998d7e3.png)

Closing. @alubbock feel free to open another issue if the problem is persisting on your hardware.
